src/Meshes.jl

```diff
@@ -37,6 +37,7 @@ import Base: sort
 import Base: ==, !
 import Base: +, -, *
 import Base: <, >, ≤, ≥
+import Base: iterate, length, IteratorSize, eltype
```

Comment:
Please define the methods in-place with `Base.iterate` without importing the names.